modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -197,6 +201,16 @@ export class Utils implements BaseUtils {
   getTxHash(txHex: string): string {
     return construct.txHash(txHex);
   }
+
+  /**
+   * Signs a given message with a given key pair
+   *
+   * @param {string} key the keypair to sign with
+   * @param {string} message the message to sign
+   */
+  signMessage(key: KeyringPair, message: string): Uint8Array {
+    return key.sign(stringToU8a(message));
+  }
```

Comment:
We spoke about this. Key pair signed message in new approach. Remove this method pls.